pkg/deployment/agent_test.go

```diff
@@ -197,33 +197,66 @@ func TestAgentCustomReporterPort(t *testing.T) {
 }
 
 func TestAgentArgumentsOpenshiftTLS(t *testing.T) {
+
```

Comment:
nit - I don't think there should be whitespace here.

Reply:
Removed.